js/index.js

```diff
@@ -1,7 +1,106 @@
 // Iteration 1: Names and Input
+// 1.1 Create a variable called "driver" with the driver's name.
+
+let driver = "Juli";
+
+// 1.2 Print in the console "The driver's name is XXXX"
+
+console.log(`The driver's name is ${driver}`);
+
+// 1.3 Create a variable called "navigator" with the navigator's name.
+
+let navigator = "Alba";
+
+// 1.4 Print in the console "The navigator's name is YYYY"
+
+console.log(`The navigator's name is ${navigator}`);
 
 
 // Iteration 2: Conditionals
+// 2.1. Depending on which name is longer, print:
+// - Driver has the longest name, it has XX characters. or
+// - Navigator has the longest name, it has XX characters. or
+// - Wow, you both have equally long names, XX characters!.
+
+if (driver.length > navigator.length){
+    console.log(`Driver has the longest name, it has ${driver.length} characters.`);
+  } else if (navigator.length > driver.length){
+    console.log(`Navigator has the longest name, it has ${navigator.length} characters.`);
+  } else {
+    console.log(`Wow, you both have equally long names, ${driver.length} characters!!!`);
+  };
+
+
+// Iteration 3: Loops
+// 3.1 Print all the characters of the driver's name, separated by a space and in capitals i.e. "J O H N"
+
+let nameCapitals;
+
+for (let i = 0; i < driver.length; i++) {
+  if (i === 0) {
+    nameCapitals = driver[i] + " ";
+  } else {
+    nameCapitals += driver[i] + " ";
+  }
+}
+
+console.log(nameCapitals.toUpperCase());
+
+// 3.2 Print all the characters of the navigator's name, in reverse order. i.e. "nhoJ"
+
+console.log(navigator.split("").reverse().join(""));
```

Comment:
Solución super inteligente! Felicitaciones. 
En este ejercicio la idea era trabajar un poco con loops. 
Otra solución podría haber sido: 
(ahora con loops, en este caso un loop de tras para adelante / reverso):

for (let i = navigator.length-1; i >= 0; i--) {
   console.log(navigator.charAt(i));
}

:-)